async/sqs/option.go

```diff
@@ -0,0 +1,72 @@
+package sqs
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/beatlabs/patron/errors"
+)
+
+const twelveHoursInSeconds = 43200
+
+// OptionFunc definition for configuring the consumer in a functional way.
+type OptionFunc func(*Factory) error
+
+// MaxMessages option for setting the max number of messages fetched.
+// Allowed values are between 1 and 10.
+func MaxMessages(maxMessages int64) OptionFunc {
```

Comment:
Isn't `int64` a bit too much for storing value ranging from 1 to 10?

Reply:
it is, but AWS uses int64 and float64. Some kind of convention I suppose.